.gitignore

```diff
@@ -16,3 +16,5 @@ build/
 dist/
 docs/_build
 venv/
+.cov2emacs.log
```

Comment:
Should this go into your global gitignore?

Reply:
good point